aten/src/ATen/native/RNN.cpp

```diff
@@ -856,6 +856,10 @@ std::tuple<io_type, Tensor, Tensor> _lstm_impl(
 
 } // anonymous namespace
 
+bool _use_cudnn_rnn_flatten_weight() {
```

Comment:
@ezyang Would this be an appropriate point for the comment that mentions that it's a cuDNN vs MIOpen thing?

Reply:
yep

Reply:
I'm sorry, I was about to put in the comment but it looks like it got merged already. :) There's always `git blame` which will lead someone to this PR :)